tools/tests/test_longmemeval_v2_official.py

```
@@ -7003,3 +7005,144 @@ def _passage_result(
+            "content": "fat state ".ljust(5_000, "f"),
+            "metadata": {"longmemeval_v2_trajectory_id": "tf"},
+        },
+        {
+            "id": "passage_small",
+            "type": "passage",
+            "content": "small slice ".ljust(400, "p"),
+            "metadata": {"longmemeval_v2_trajectory_id": "tp"},
+        },
+    ]
+
+    starved, starved_meta = module.compile_operational_evidence_set(
```

Ruff (0.16.1)

[warning] Unpacked variable `starved_meta` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)